packages/nuxt/src/head/runtime/composables.ts

```diff
@@ -12,3 +13,18 @@ import { useNuxtApp } from '#app'
 export function useHead<T extends HeadAugmentations> (input: UseHeadInput<T>, options?: HeadEntryOptions): ActiveHeadEntry<UseHeadInput<T>> | void {
   return useNuxtApp()._useHead(input, options)
 }
+
+/**
+ * The `useSeoMeta` composable lets you define your site's SEO meta tags
+ * as a flat object with full TypeScript support.
+ *
+ * This helps you avoid typos and common mistakes, such as using `name`
+ * instead of `property`.
+ *
+ * This function will have no effect (and will return nothing) if called on the client.
+ */
+export const useSeoMeta: typeof _useSeoMeta = (meta) => {
+  if (process.server) {
```

Comment:
I think we need to bypass this at least when global ssr is disabled with `ssr:false`.

Reply:
Why would we want to run this on client-side when `ssr: false`?

Reply:
Because public websites still get rendered crawled by search engines and benefit from seo meta as such. Obviously not best practice but a valid deployment option with Nuxt.

Reply:
Yes, and needs to be a build-time flag to work with tree-shaking. That's why I'm suggesting relying on global `ssr: false` (can be passed similar to `process.server`). global `ssr: false` mode is different.

`useServerSeoMeta` seems good idea too!